app/console

```diff
@@ -15,7 +15,6 @@ set_time_limit(0);
  * @var Composer\Autoload\ClassLoader $loader
  */
 $loader = require __DIR__.'/../app/autoload.php';
```

Comment:
This is not part of this pull request, additionally I believe the variable is kept so that you can uncomment the code below: https://github.com/symfony/symfony-standard/blob/2.8/web/app.php#L11-L19